tox.ini

```diff
@@ -42,7 +42,8 @@ show-source = True
 max-line-length = 100
 exclude = venv,.git,.tox,dist,*egg,cachito/web/migrations,.env
 # W503 line break before binary operator
-ignore = D100,D104,D105,W503
+# E203 whitespace before ':'
+ignore = D100,D104,D105,W503,E203
```

Comment:
The reason this is not disabled inline, is that black will also deal with the same issue. Is this right? If so, can we add a comment on that?

Reply:
Exactly, added comment